optuna/notebooks/optuna_rapids.ipynb

```diff
@@ -13,28 +13,28 @@
     "## Dask-Optuna\n",
```

Comment:
<p>Should make this a relative path. Not many users can store in /home/data</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cloud-ml-examples/pull/45/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>